README.md

```diff
@@ -54,6 +54,10 @@ Confirmed working boards:
 - OpenShock (Legacy)
   - Core V1
 
+Experimental Boards:
+
+- DFRobot Firebeetle ESP32-E
```

Comment:
Perhaps maybe just put this under with the rest of the boards since it reallly isn't that expirimental.

Reply:
yeah, that should be fine 😄